java/src/jmri/jmrix/can/cbus/CbusReporter.java

```diff
@@ -38,6 +38,22 @@ public class CbusReporter extends AbstractRailComReporter implements CanListener
     private final TrafficController tc; // can be removed when former constructor removed
     private final CanSystemConnectionMemo _memo;
 
+    /**
+     * All CbusReporters should clear themselves after a timeout.
```

Comment:
I think it would be better if this sentence is a question.
```
Should all CbusReporters clear themselves after a timeout?
true = yes, false = no
```